lcls-twincat-optics/lcls_twincat_optics_plc/DUTs/ENUM_Coating_States.TcDUT

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <DUT Name="ENUM_Coating_States" Id="{118e1a24-82d2-4598-99ea-540bbe0bbd1a}">
+    <Declaration><![CDATA[{attribute 'qualified_only'}
+TYPE ENUM_Coating_States :
+(
+	Unknown := 0,
+    Coating1 := 1,
+    Coating2 := 2
```

Comment:
I think we need to specify the coating types in the Enum so that the combobox shows the coating types

Reply:
This is specified in the function block. I am not sure they are all the same for all the HOMS mirrors.

Reply:
Wouldn't the combo box pick from the sCoating1Type?

Reply:
I think the pytmc pragma needs to specify it if it's not the same as the enum strings
If the homs mirrors have different coatings then we need to either have several different enums or a way to override the string fields per instance when we make the PVs

Reply:
OK got it. Now I need to do some thinking. Maybe we don't have to put it in the lib but in every mirrors implementation. I thought it will be neater in the lib.

Reply:
Agreed, neater in the lib. Maybe there's a pytmc trick we can use to filter down the enum strings for different mirror coatings @klauer @n-wbrown

Reply:
I can try to come up with something here, but sadly I don't think there is an easy way currently.

Reply:
Thank you all for your feedback. I will mark this pr as invalid and will work on another one now that I have a better understanding of how the state names are showing in epics.